### DISCUSSION TITLE
Yaşam, Evren ve Herşey

### DISCUSSION THREAD

**Discussion by slos8888** [sonsuzus/sonsuzus#434] [Soru Cevap]
Problem:
Programınız Yaşamın, Evrenin ve Her Şeyin Cevabını bulmak için kaba kuvvet yaklaşımını kullanmaktır. Daha doğrusu... küçük sayıları input kısmından output kısmına yeniden yazın. 42 sayısını okuduktan sonra giriş işlemini durdurun. Girişteki tüm sayılar bir veya iki basamaklı tam sayılardır.

Link:https://www.hackerearth.com/practice/basic-programming/input-output/basics-of-input-output/practice-problems/algorithm/life-the-universe-and-everything/

**Comment by sonsuzus** [comment]
Bu soru daha önce eklendi. Python sorusu diye özel bir durum yok, programlama dili serbest istediğin dilde çözebiliyorsun.

**Reply by slos8888** [reply]
Tamamdır hocam görmemişim bugün eklerim yeni bir tane
